terraform/lambda_opt_out/variables.tf

```diff
@@ -0,0 +1,110 @@
+variable "policy_name" {
+  description = "Name of the IAM policy"
+  type = string
+}
+variable "policy_description" {
+  description = "Description of the IAM policy"
+  type = string
+}
+variable "iam_role_name" {
+  description = "Name of the IAM role"
+  type   =  string
+}
+variable "key_description" {
+  description = "Description for the KMS key"
+  type = string
+}
+variable "enable_key_rotation" {
+  description = "Enable automatic key rotation"
+  type =  bool
+}
```

Comment:
Reduce the number of variables as much as possible by removing any that will be the same in all target environments. The enable_key_rotation variable, for example, can be removed because we will always enable key rotation.